dali/pipeline/data/buffer.h

```diff
@@ -37,6 +37,22 @@ namespace dali {
 class GPUBackend;
 class CPUBackend;
 
+/**
+ * @brief State of a batch object. Can be used to pin the state to one of the two contiguous /
```

Comment:
`Can be used to force` or `Can be used set` - word pin has different connotation and it is event used in same sentence with different meaning.

Reply:
Done